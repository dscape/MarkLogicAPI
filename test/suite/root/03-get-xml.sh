#!/bin/sh
curl -sH "Accept: application/xml" localhost:6984