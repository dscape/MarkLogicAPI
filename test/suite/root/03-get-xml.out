<welcome>
  <product>marklogic</product>
  <version>4.2-1</version>
</welcome>
 